app/views/providers/check_benefits/index.html.erb

```diff
@@ -1,5 +1 @@
-<% if should_use_ccms? %>
-  <%= render 'use_ccms' %>
-<% else %>
-  <%= render 'check_benefits_result' %>
-<% end %>
+<%= render 'check_benefits_result' %>
```

Comment:
Can we remove the partial and move the content into this template?

As far as I can tell the partial isn't used anywhere else.